bpf/lib/process.h

```diff
@@ -564,8 +564,13 @@ FUNC_INLINE struct execve_info *execve_joined_info_map_get(__u64 tid)
 _Static_assert(sizeof(struct execve_map_value) % 8 == 0,
 	       "struct execve_map_value should have size multiple of 8 bytes");
 
+#define SENT_FAILED_UNKNOWN 0 // unknown error
+#define SENT_FAILED_EBUSY   1 // EBUSY
+#define SENT_FAILED_ENOSPC  2 // ENOSPC
+#define SENT_FAILED_MAX	    3
+
 struct kernel_stats {
-	__u64 sent_failed[256];
+	__u64 sent_failed[256][SENT_FAILED_MAX];
 };
```

Comment:
just a note.. I think it's ok because it's still small map, but it is per cpu and this change multiplies the memory usage by 3